webservice/webservice.py

```diff
@@ -322,7 +326,64 @@ def get_manifest():
     # Get the response back
     logger.info("Creating the API response")
     response = es_td.transform_manifest(filters=filters)
-    # Return the excel file
+    if output_formats == 'tsv':  # returns metadata as TSV file
```

Comment:
You should default to tsv, i.e., if the parameter is not specified, you should assume it's tsv. Otherwise you will break existing code that it calling this method without the query parameter.